player_alsa.h

```diff
@@ -0,0 +1,33 @@
+#include <alsa/asoundlib.h>
+
+void check(int *err, int newErr) {
```

Comment:
Or, I'd include this file into the Go file as a comment (this is not mandatory).